src/OpenLoco/src/Windows/ObjectSelectionWindow.cpp

```diff
@@ -313,8 +313,7 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
         _tabObjectList.reserve(objects.size());
         for (auto [index, object] : objects)
         {
-            auto entry = TabObjectEntry{ index, object, true };
-            _tabObjectList.emplace_back(entry);
+            _tabObjectList.emplace_back(index, object, true);
```

Comment:
@duncanspumpkin I'm not sure why this doesn't work. Seems analogous to your example in https://godbolt.org/z/nbP5KzMac

Reply:
I think it might be because index and object are references. If you did std::move(index), std::mvoe(object) it would work. hmm all a bit messy now. The other alternative is `_tabObjectList.emplace_back(TabObjectEntry{index, object, true})` or just revert to the old style and use (std::move and emplace) or push_back

Reply:
Went with a revert and `std::move` on `entry`.